### PR TITLE
inflow layer properties diagnostic addition

### LINKED ISSUE #3: Inflow properties test
This is example text for an issue to add a new diagnostic to identify elevated convection. This is a sample text for the inflow layer properties diagnostic for demonstration purposes for the RMED Team Brief in January 2024.

### PR DESCRIPTION
**Pull request by daflack**
fixes #3

### CODE REVIEW

File: src/CSET/operators/convection.py [hunk 1]
```diff
@@ -136,3 +137,107 @@ def cape_ratio(SBCAPE, MUCAPE, MUCIN, MUCIN_thresh=-75.0):
...
+    time and ensemble member.`` these warnings are expected when the orography files
+    are not 2-dimensional, and do not cause any problems.
+
+    References
+    ----------
+    .. [1] Thompson, R. L. Mead, C. M., and Edwards, R., (2007) "Effective
+       Storm-Relative Helicity and Bulk Shear in Supercell Thunderstorm
+       Environments." Weather and Forecasting, vol. 22, 102-115,
+       doi: 10.1175/WAF969.1
+    .. [2] Flack, D.L.A., Lehnert, M., Lean, H.W., and Willington, S. (XXXX)
```
Comment (jwarner8) [inline src/CSET/operators/convection.py:208]: Should this be the ref from 2023?

**Review by jwarner8 — APPROVED** [review]
Looks good, needs minor edit to reference

File: src/CSET/operators/convection.py [hunk 2]
```diff
@@ -136,3 +137,107 @@ def cape_ratio(SBCAPE, MUCAPE, MUCIN, MUCIN_thresh=-75.0):
...
+    # layer height to 1 implying elevated convection.
+    EC_Flagd[EIB.data > (BLheight.data + Orography.data)] = 1.0
+    # Take the coordinates from an existing cube and replace the data.
+    inflow_properties_cube = EIB.copy()
+    inflow_properties_cube.data = EC_Flagd
+    # Rename and remove STASH code.
+    inflow_properties_cube.var_name = "inflow_layer_properties"
+    inflow_properties_cube.attributes.pop("STASH", None)
+    # Return the cube.
+    return EC_Flagd
```
Comment (jfrost-mo) [inline src/CSET/operators/convection.py:243]:
```suggestion
    return inflow_properties_cube
```

Is `EC_Flagd` really what you wanted to return?

**Review by jfrost-mo — CHANGES_REQUESTED** [review]
A single major issue identified, should be easily fixed, but otherwise looks good.

File: src/CSET/operators/convection.py (hunk 1 above)
Comment (daflack) [inline src/CSET/operators/convection.py:208]:
```suggestion
    .. [2] Flack, D.L.A., Lehnert, M., Lean, H.W., and Willington, S. (2023)
```

**Review by jfrost-mo — APPROVED** [review]
Looks good to me. :)

**Pull request merged**